counter/src/main/java/org/infinispan/counter/impl/manager/CounterConfigurationStorage.java

```diff
@@ -28,6 +28,13 @@
     */
    void store(String name, CounterConfiguration configuration);
 
+   /**
+    * Remove a counter configuration
```

Comment:
does this remove the counter or the configuration of the counter ?

Reply:
It just removes the configuration